### ISSUE TITLE
Numeric editor with dropdown does not provide aria-expanded=false attribute initially

### ISSUE THREAD

**Issue by dkamburov** [IgniteUI/ignite-ui#2106]
## Description  
Numeric editor with dropdown does not provide aria-expanded=false attribute.

 * ignite-ui version:
 * browser: 

## Steps to reproduce  

1. Open http://jsfiddle.net/dkamburov/fd3v8sxp/
2. Inspect the first numeric editor, before expanding the dropdown

## Result  
There is no aria-expanded=false attribute

## Expected result  
There should be aria-expanded=false attribute

## Attachments  
![image](https://user-images.githubusercontent.com/1182001/105059389-e38dce80-5a7f-11eb-84b0-8cc8fcd01ac0.png)

**Comment by dkamburov** [comment]
Also it seems that ARIA 1.0 is used and if you consider updating to ARIA 1.1.

Note: The primary difference between the ARIA 1.0 pattern and the ARIA 1.1 pattern is the placement of the combobox role.
Note: In the ARIA 1.1 combobox pattern, the combobox uses aria-controls instead of aria-owns.
https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.0pattern/combobox-autocomplete-both.html
